_styles/main.css

```diff
@@ -942,6 +942,79 @@ input:focus:-ms-input-placeholder {
     margin: 6px !important;
 }
 
+@media only screen and (max-width: 425px) {
+    .dialog {
+        width: calc(100vw - 32px);
+    }
+}
+
+.dialog {
+    background-color: #f5f5f5;
+    border-radius: 3px;
+    box-shadow:
+        inset 1px 0 0 0 rgba(255, 255, 255, 0.2),
+        inset -1px 0 0 0 rgba(255, 255, 255, 0.2),
+        inset 0 1px 0 0 #fff,
+        0 0 0 1px rgba(0, 0, 0, 0.2),
+        0 8px 20px 1px rgba(0, 0, 0, 0.14),
+        0 3px 28px 2px rgba(0, 0, 0, 0.12),
+        0 5px 10px -3px rgba(0, 0, 0, 0.4);
+    margin: 0 auto;
+    padding: 12px;
+    padding-top: 24px;
+}
+
+.dialog img {
+    display: inline-block;
+    margin-right: 12px;
+    -webkit-user-select: none;
+    vertical-align: top;
+}
+
+.dialog .content-area {
+    display: inline-block;
+    width: calc(100% - 48px - 18px); /* Subtract icon width and right margin */
+}
+
+.dialog p {
+    line-height: 24px;
+    margin-top: 0;
+    margin-bottom: 6px;
+    max-width: 380px;
+}
+
+.dialog p.primary {
+    font-weight: 700;
+}
+
+.dialog .action-area {
+    display: -ms-flexbox;
+    display: flex;
+    -ms-flex-flow: row;
+    flex-flow: row;
+    -ms-flex-pack: end;
+    flex-wrap: wrap;
+    justify-content: flex-end;
+    margin-top: 12px;
```

Comment:
also remove all the browser prefixes here. we automatically add them during build.